src/models/data.h

```diff
@@ -46,23 +46,40 @@ QString prettifySymbol(const QString& symbol);
 
 struct Symbol
 {
+    static bool prettifySymbol;
+
     Symbol(const QString& symbol = {}, const QString& binary = {}, const QString& path = {})
         : symbol(symbol)
-        , prettySymbol(prettifySymbol(symbol))
         , binary(binary)
         , path(path)
     {
     }
 
     // function name
     QString symbol;
-    // prettified function name
-    QString prettySymbol;
+    // prettified function name (cached in getSymbol)
+    mutable QString cachedPrettySymbol;
```

Comment:
considering that most symbols will be prettified by default, I think it would be OK and good to always prettify them on construction of this struct